rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2471,6 +2499,11 @@ void RSXThread::End()
 {
 	Draw();
 
+	if (Ini.RSXFrameStep.GetValue())
```

Comment:
Frame stepping. After each frame is drawn, it pauses the emulator, so you can see the result of each frame drawn and then distinguish the RSX command proportion, that caused the unexpected result on the screen. It's mainly meant to be used alongside RSX logging. This is useful for example in cases where something is drawn on the screen, but dissapears after a certain amount of frames, as so happens to be in a couple SDK graphics samples.